images/pegasus/scripts/startup.sh

```diff
@@ -0,0 +1,136 @@
+gow_log "Pegasus - Configure"
+mkdir -p "$PEGASUS_CFG"
+gow_log "Pegasus - Copying game_dirs.txt, if not edited"
+cp -u /cfg/app/game_dirs.txt "${PEGASUS_CFG}/game_dirs.txt"
+
+#########################################
+# Configure PCSX2
+#########################################
+PCSX2_CFG=$HOME/.config/PCSX2
+
+gow_log "PCSX2 - Configure"
+mkdir -p "$PCSX2_CFG/inis/"
+cp -u /cfg/PCSX2/PCSX2.ini "${PCSX2_CFG}/inis/PCSX2.ini"
+
+#########################################
+# Configure Dolphin
+#########################################
+DOLPHIN_CFG=$HOME/.config/dolphin-emu
+
+mkdir -p "$DOLPHIN_CFG"
+gow_log "Dolphin - Copying config, if not edited"
+cp -u /cfg/dolphin/Dolphin.ini "$DOLPHIN_CFG/Dolphin.ini"
+
+#########################################
+# Configure Retroarch
+#########################################
+RETROARCH_CFG_DIR=$HOME/.config/retroarch
+
+# Copying config in case it's the first time we mount from the host
+gow_log "Retroarch - Copying config, if not edited"
+mkdir -p "$RETROARCH_CFG_DIR/cores/"
+cp -u /cfg/retroarch/retroarch.cfg "$RETROARCH_CFG_DIR/retroarch.cfg"
+
+gow_log "Retroarch - Checking RA Assets presence, if none - install them"
+if [ ! -d "$RETROARCH_CFG_DIR/assets" ]; then
+    gow_log "Retroarch - No assets found, starting install"
+    wget -q -P /tmp https://buildbot.libretro.com/assets/frontend/assets.zip
+    7z x /tmp/assets.zip -bso0 -bse0 -bsp1 -o"$RETROARCH_CFG_DIR/assets"
+    rm /tmp/assets.zip
+fi
+
+# Copy pre-installed cores from the retroarch ppa
+# shellcheck disable=SC2046
+# cp -u /usr/lib/$(uname -m)-linux-gnu/libretro/* "$CFG_DIR/cores/"
+
+#########################################
+# Configure Yuzu
+#########################################
+YUZU_CFG_DIR=$HOME/.local/share/yuzu
+
+gow_log "YUZU - Copying pred/title keys for YUZU"
+if test -f /bioses/prod.keys; then
+    gow_log "YUZU - /bioses/prod.keys present, copy to YUZU folder"
+	  mkdir -p $YUZU_CFG_DIR/keys/
+    cp -u $HOME/bioses/prod.keys $YUZU_CFG_DIR/keys/prod.keys
+fi
+if test -f /bioses/prod.keys; then
+    gow_log "YUZU - /bioses/title.keys present, copy to YUZU folder"
+	  mkdir -p $YUZU_CFG_DIR/keys/
+    cp -u $HOME/bioses/title.keys $YUZU_CFG_DIR/keys/title.keys
+fi
+
+#########################################
+# Configure CEMU
+#########################################
+CEMU_CFG_DIR=$HOME/.config/Cemu
+
+gow_log "CEMU - Setting default sound device and run in fullscreen, if not edited"
+cp -u /cfg/cemu/settings.xml /cfg/cemu/settings_new.xml
+searchString="<TVDevice>replace_me</TVDevice>"
+replaceString="<TVDevice>${PULSE_SINK}</TVDevice>"
+sed -i -e "s|$searchString|$replaceString|g" /cfg/cemu/settings_new.xml
+searchString="<PadDevice>replace_me</PadDevice>"
+replaceString="<PadDevice>${PULSE_SINK}</PadDevice>"
+sed -i -e "s|$searchString|$replaceString|g" /cfg/cemu/settings_new.xml
+# Set the date of the file back to the original date
+touch -r /cfg/cemu/settings.xml /cfg/cemu/settings_new.xml
```

Comment:
I get a permission denied on my side when doing this, `/cfg/` has been created by `root` in the Dockerfile; in the startup file we should only modify `$HOME`

Reply:
Fixed it!